python/ray/experimental/serve/api.py

```diff
@@ -65,7 +66,10 @@ def init(kv_store_connector=None,
          blocking=False,
          http_host=DEFAULT_HTTP_HOST,
          http_port=DEFAULT_HTTP_PORT,
-         ray_init_kwargs={"object_store_memory": int(1e8)},
+         ray_init_kwargs={
+             "object_store_memory": int(1e8),
+             "num_cpus": max(cpu_count(), 8)
```

Comment:
Is it supposed to stay here?

Reply:
Yes, serve requires 3+ actors and ray by default starts `cpu_count()` actors. So this really to add a lower bound to num_cpus to 8